src/js/tech/html5.js

```diff
@@ -282,6 +293,33 @@ class Html5 extends Tech {
     this.textTracks().removeTrack_(e.track);
   }
 
+  removeOldTextTracks() {
+    // This will loop over the texttracks and check if they are still used
+    // If not, they will be removed from the emulated list
+    let removeTracks = [];
+
+    for (let i = 0; i < this.textTracks().length; i++) {
+      let techTrack = this.textTracks()[i];
```

Comment:
we probably should cache `this.textTracks()` in a variable outside the forloops so we don't have to make the look up each time.